.travis.yml

```diff
@@ -7,7 +7,7 @@ install:
 script:
   - xbuild /p:Configuration=Release NRasterizer.sln
   - mono ./testrunner/NUnit.ConsoleRunner.3.5.0/tools/nunit3-console.exe ./NRasterizer.Tests/bin/Release/NRasterizer.Tests.dll
-  - nuget pack NOpenType/NOpenType.nuspec
+  - nuget pack NOpenType/NOpenType.nuspec -NoDefaultExcludes
```

Comment:
not sure why but without this nuget.exe would silently ignore the netstandard line in the nuspec.